src/transformState.js

```diff
@@ -5,7 +5,35 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
+        removeProperties(action, state);
```

Comment:
Would be nice to add some empty lines between different cases to make your switch more readable
```suggestion
    switch (action.type) {
      case 'addProperties':
        Object.assign(state, action.extraData);
        break;
      
      case 'removeProperties':
        removeProperties(action, state);
```